### PR TITLE
Add an option to show a diff in the commit

### LINKED ISSUE #38: Diff while commiting
When creating a commit, it's handy to see at a glance what you are actually committing, a diff displayed at the bottom is the optimal solution.

Should also be optional, as it would probably add extra load time on larger commits.

### PR DESCRIPTION
**Pull request by p3lim**
Still needs a update to the syntax, but it's a start.
Resolves #38.

Note; I'm new to Python.

### CODE REVIEW

File: core/commands/commit.py [hunk 1]
```diff
@@ -62,6 +65,10 @@ def run(self, edit):
         else:
             initial_text = COMMIT_HELP_TEXT
 
+        if self.view.settings().get("git_savvy.commit_view.diff"):
```
Comment (divmain) [inline core/commands/commit.py]:
```python
if sublime.load_settings("GitSavvy.sublime-settings").get("show_commit_diff"):
```

File: core/commands/commit.py [hunk 2]
```diff
@@ -33,10 +34,12 @@ def run(self, **kwargs):
 
     def run_async(self, repo_path=None, include_unstaged=False, amend=False):
         repo_path = repo_path or self.repo_path
+        show_diff = sublime.load_settings("GitSavvy.sublime-settings").get("show_commit_diff")
```
Comment (divmain) [inline core/commands/commit.py]: Delete.

File: core/commands/commit.py [hunk 3]
```diff
@@ -33,10 +34,12 @@ def run(self, **kwargs):
...
         view = self.window.new_file()
         view.settings().set("git_savvy.get_long_text_view", True)
         view.settings().set("git_savvy.commit_view.include_unstaged", include_unstaged)
         view.settings().set("git_savvy.commit_view.amend", amend)
+        view.settings().set("git_savvy.commit_view.diff", show_diff)
```
Comment (divmain) [inline core/commands/commit.py]: Delete.

**Comment by divmain** [comment]
Thanks @p3lim!  Please make the requested updates, and I'll merge this in.  I plan to follow-up with #50 and #51.

**Comment by alextegelid** [comment]
When I'm writing the commit message I get the "GitSavvy Make Commit" syntax which doesn't really affect anything but the @ character and the comments. Is this intentional or is something wrong? If I choose the "GitSavvy Commit" I get a better syntax highlighting.
screen

(Also, thanks for an awesome plugin!)

**Comment by p3lim** [comment]
@alextegelid see #50.

**Comment by alextegelid** [comment]
I see, thanks! I thought it had already been implemented

**Pull request merged**